docs/wiki/[User-Guide]-Upgrade-from-v3.3.0-to-v4.0.0.md

```diff
@@ -5,7 +5,7 @@ The `v4.0.0` release is predominantly a policy refresh release that brings the l
 
 ### New features
 
-- Support for version 3.54.0 of the `azurerm` provider. This is a breaking change and will require updates to your calling module.
+- Support for version 3.74.0 of the `azurerm` provider. This is a breaking change and will require updates to your calling module.
```

Comment:
This should not changes as it is a historical record. 

I think this would need to be a major version update, so a v4.2.0 to v5.0.0 file is probably required. This is because it will break for existing users targeting 3.54.0 in their root module.

Reply:
Yeah I think so
We will also merge #806 before v5